canvas_modules/common-canvas/src/common-properties/components/control-item/control-item.jsx

```diff
@@ -86,6 +86,12 @@ class ControlItem extends React.Component {
 					{indicator}
 					{tooltip}
 				</div>);
+		} else {
+			// Provide a label for controls with hidden labels to support accessibility
+			label = (
+				<div className={classNames("properties-label-container-hidden")}>
+					<label className="properties-control-label-hidden">{this.props.control.label.text}</label>
```

Comment:
We might want `this.props.control?.label?.text` in case the label wasn't provided.

Reply:
Included this in the PR. Thanks.